lib/varip.c

```diff
@@ -0,0 +1,73 @@
+#include <rte_ip.h>
+#include <rte_mbuf.h>
+
+#include "gatekeeper_varip.h"
+
+struct ipv6_opt_hdr {
+	uint8_t nexthdr;
+	uint8_t hdrlen;
+} __attribute__((packed));
+
+#define ipv6_optlen(p) (((p)->hdrlen + 1) << 3)
+
+int
+ipv6_skip_exthdr(const struct ipv6_hdr *ip6hdr, int remaining_len,
+	int start, uint8_t *nexthdrp)
```

Comment:
The value passed in for `start` is always `sizeof(struct ipv6_hdr)`. Can we get rid of this parameter and just declare `start` on the first line of the function?